### PR TITLE
Added Address class

### PR DESCRIPTION
**Pull request by MukeshGuptaDev**
I added address class

### CODE REVIEW

File: src/utility/address.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  return state_;
+}
+
+int Address::getPincode(){
+  return pincode_;     
+}
+
+std::string Address::getAddress(){
+  std::string result = location_;
+      result += "," + district_;
```
Comment (TheMartian0x48) [inline src/utility/address.cpp:55]: align statements

File: src/utility/address.hpp [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+class Address{
+  private:
+    std::string location_, district_, state_, country_;
+    int pincode_;
+    
+  public:
+    Address(std::string location, std::string district, 
+    std::string state, std::string country, int pincode);
+
```
Comment (TheMartian0x48) [inline src/utility/address.hpp:14]: no space between two method declaration

File: src/utility/address.cpp (hunk 1 above)
Comment (MukeshGuptaDev) [reply src/utility/address.cpp:55]: i modified allign statement

File: src/utility/address.hpp (hunk 2 above)
Comment (MukeshGuptaDev) [reply src/utility/address.hpp:14]: i modified it.

File: src/utility/address.cpp (hunk 1 above)
Comment (TheMartian0x48) [reply src/utility/address.cpp:55]: there must be 2 space from left, not 4
please re-format

File: src/utility/address.hpp [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    void setDistrict(std::string district);
+    void setState(std::string state);
+    void setCountry(std::string country);
+    void setPincode(int pincode);
+    std::string getLocation();
+    std::string getCountry();
+    std::string getDistrict();
+    std::string getState();
+    int getPincode();
+
```
Comment (TheMartian0x48) [inline src/utility/address.hpp:25]: no space between two method declaration

**Pull request merged**